common/src/main/resources/reference.conf

```diff
@@ -11,6 +11,7 @@ akka {
 
           journal-collection = "akka_persistence_journal"
           journal-index = "akka_persistence_journal_index"
+          journal-seq-nr-index = "akka_persistence_journal_seq_nr_index"
```

Comment:
I'll need to default the name to what has already been published, but at least the configuration will let you override this.